Library/Homebrew/cmd/vendor-install.sh

```diff
@@ -20,6 +20,11 @@ then
 elif [[ -n "$HOMEBREW_LINUX" ]]
 then
   case "$HOMEBREW_PROCESSOR" in
+    armv[6789]*)
```

Comment:
Please put this under `x86_64`

Reply:
Also, is `arm8` or `arm9` returned by any Linux distros currently? If not: let's just make this `[67]`

Reply:
I don't know the answer to this question. I've changed it to `arm[67]*`.